### PR TITLE
Laracon Summary Day Three

### PR DESCRIPTION
**Pull request by unicodeveloper**
laracon-summary-day-three

### CODE REVIEW

File: _posts/2016-07-30-laracon-summary-day-3.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+{% include tweet_quote.html quote_text="40% of users abandon a website that takes more than 3 seconds to load - Fabien Potencier" %}
+
+**[Chuck Reeves](https://twitter.com/manchuck)** gave a talk on *Practical Software Estimation*. Chuck dropped nuggets of wisdom on how to provide estimates for a software project. He emphasized why developers should estimate because they are the ones writing the code. Chuck compared developers to cardiologists and gave an example of how the latter cannot be coerced into performing a surgical operation for five hours instead of eight. As a developer, if it takes you eight hours to complete a task, do not negotiate for less! Check out the slides [here](http://www.slideshare.net/manchuck/stop-multiplying-by-4-laracon).
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">No estimation talk ever will be complete without Bayes Theorem. When you get new info, the level of uncertainty will decrease <a href="https://twitter.com/hashtag/laracon?src=hash">#laracon</a></p>&mdash; Michael Dyrynda (@michaeldyrynda) <a href="https://twitter.com/michaeldyrynda/status/759057447898054656">July 29, 2016</a></blockquote>
```
Comment (sebadoom) [inline _posts/2016-07-30-laracon-summary-day-3.markdown]: Link to Bayes Theorem?

File: _posts/2016-07-30-laracon-summary-day-3.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## Conclusion
+
+Laracon US 2016 is actually the first Laravel Conference to be 95 percent technical talks. All the speakers were great and the talks were educational and insightful.
+
+If you are just getting started in developing Laravel 5 apps, check out our [Laravel app tutorial](https://auth0.com/blog/2016/06/23/creating-your-first-laravel-app-and-adding-authentication/). Need an alternative means of adding authentication and authorization to your Laravel apps? Sign up for a free Auth0 account and get your user authentication set up in few minutes using our [Laravel quickstarts](https://auth0.com/docs/quickstart/webapp/laravel). Building an API using Laravel and wondering how to add authentication using JWT? Auth0 has got you covered. Check out our [Laravel API quickstarts](https://auth0.com/docs/quickstart/backend/php-laravel) to implement that in less than 10 minutes.
```
Comment (sebadoom) [inline _posts/2016-07-30-laracon-summary-day-3.markdown]: The sign-up text should open the sign-up screen. Set it as `<a href="javascript:signup()">Sign up</a>`.

File: _posts/2016-07-30-laracon-summary-day-3.markdown [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+design:
+  bg_color: "#4a4a4a"
+  image: "https://cdn.auth0.com/blog/laravel-auth/logo.png"
+tags:
+- Laravel
+- Laravel-5.3
+- Laracon
+- LaraconUS
+---
+
```
Comment (sebadoom) [inline _posts/2016-07-30-laracon-summary-day-3.markdown]: Related posts?

File: _posts/2016-07-30-laracon-summary-day-3.markdown (hunk 1 above)
Comment (unicodeveloper) [inline _posts/2016-07-30-laracon-summary-day-3.markdown]: This is an embedded tweet, so i can't put the link to Bayes Theorem :smile:

**Pull request merged**